tumblr-gif-search/app.py

```diff
@@ -7,16 +7,19 @@
 
 from jina.flow import Flow
 
-# num_docs = os.environ.get('MAX_DOCS', 30)
-
 GIF_BLOB = 'data/*.gif'
+LEN_DOCS = len(glob(GIF_BLOB))
+# allows for returning all docs
 JINA_TOPK = 5000
```

Comment:
this will need to be changes when we index a good amount of docs

Reply:
From what I've seen in multires, there's no attempt to limit to TOPK. It does seem like it's not really working with the chunking of the gifs. Should we just remove any mention of it from the example?